packages/vitest/src/node/errors.ts

```diff
@@ -13,3 +13,19 @@ export class GitNotFoundError extends Error {
     super('Could not find Git root. Have you initialized git with `git init`?')
   }
 }
+
+export class IncludeTaskLocationDisabledError extends Error {
+  code = 'VITEST_INCLUDE_TASK_LOCATION_DISABLED'
+
+  constructor() {
+    super('Recieved line number filters while `includeTaskLocation` option is disabled')
+  }
+}
+
+export class RangeLocationFilterProvidedError extends Error {
+  code = 'VITEST_RANGE_LOCATION_FILTER_PROVIDED'
+
+  constructor(filter: string) {
+    super(`Location filters are not allowed: ${filter}`)
```

Comment:
This error is thrown when `-` is used in the location, but it doesn't explain why it's wrong or how to fix it. Maybe add more explanation?